tdrs-frontend/src/components/Reports/Reports.jsx

```diff
@@ -62,11 +68,27 @@ function Reports() {
     const reportHeader = `${currentStt} - Fiscal Year ${selectedYear} - ${quarters[selectedQuarter]}`
 
     if (form.length === 3) {
+      // Hide upload sections while submitting search
       if (isUploadReportToggled) {
         setIsToggled(false)
       }
 
+      // Clear existing file list from state to ensure fresh results
+      dispatch(clearFileList())
+
+      // Retrieve the files matching the selected year and quarter.
+      dispatch(
+        getAvailableFileList({
```

Comment:
This was moved from an effect in [UploadReport.jsx](https://github.com/raft-tech/TANF-app/pull/1061/files#diff-8829eb3655bfda6ea843b0085e91c02a9aedcdf0f860cedaea72ba3bea519568L41-L43) to ensure that files shown in the form only change when the search button is pressed and not on any time the selections are changed without hitting seaarch.